demos/browser/app/meetingV2/styleV2.scss

```diff
@@ -797,3 +798,38 @@ a.markdown:active {
   outline: none;
   background: none;
 }
+
+.stats-info {
+  position: absolute;
+  background-color: rgba($color: #000000, $alpha: 0.5);
```

Comment:
Where did you get this color? Can we re-use one that we already defined?

Reply:
I needed some transparency between the actual video and the over stats hence used the `rgba` which gives helps us with `alpha` value, the existing colors do not use and match it, hence used it. Is that fine?